tools/codegen/codegen.py

```diff
@@ -76,6 +76,10 @@ def re_split_function(regex, text):
     return re.split(regex, text)
 
 
+def re_replace_function(text, match, sub):
```

Comment:
less cognitive dissonance if we call it `re_sub_function` and `re_sub` is the filter name below.

Reply:
good suggestion, done.